### PR TITLE
feat: improving loginAttempt storage

### LINKED ISSUE #1031: Possibility of bypassing the PIN lock mechanism
### Problem 
It was identified that the number of incorrect PIN attempts is saved in the application's database, thus making it possible to bypass this control by executing an update on the database.

### Impact
An ill-intentioned user can bypass the PIN lock mechanism and execute a brute-force attack for PIN discovery.

### Evidence
image-2023-11-16-18-10-00-218

### References
https://owasp.org/www-community/controls/Blocking_Brute_Force_Attacks

### PR DESCRIPTION
**Pull request by CarolineLCa**
# Summary of Changes

Fix issue storing `loginAttempt` on `keychain`

# Related Issues

Resolves  #1031 

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [X] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [X] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [X] Updated documentation as needed for changed code and new or modified features;
- [X] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Comment by CarolineLCa** [comment]
@wadeking98  @jleach @bryce-mcmath may you take a look, please?

**Comment by bryce-mcmath** [comment]
@wadeking98 can you check this one out when you have a moment?

File: packages/legacy/core/App/services/keychain.ts [hunk 1]
```diff
@@ -95,6 +103,19 @@ export const loadWalletSalt = async (): Promise<WalletSalt | undefined> => {
   return JSON.parse(result.password) as WalletSalt
 }
 
+export const loadLoginAttempt = async (): Promise<LoginAttempt | undefined> => {
```
Comment (bryce-mcmath) [inline packages/legacy/core/App/services/keychain.ts:106]: Can you export this method from the bifold index?

Comment (CarolineLCa) [reply packages/legacy/core/App/services/keychain.ts:106]: @bryce-mcmath  I didn't understand. We don't have this method in the index

Comment (wadeking98) [reply packages/legacy/core/App/services/keychain.ts:106]: if you go into `packages/legacy/core/App/index.ts` and import it and export it from there then other wallets can access it, like BC Wallet. Other than that it looks good

File: packages/legacy/core/App/index.ts [hunk 2]
```diff
@@ -144,4 +145,5 @@ export {
   components,
   contexts,
   Text,
+  loadLoginAttempt,
```
Comment (CarolineLCa) [inline packages/legacy/core/App/index.ts:148]: @wadeking98  Is this okay?

**Review by wadeking98 — APPROVED** [review]

**Review by thiagoromanos — APPROVED** [review]

**Pull request merged**